cmd/etcd/etcd.go

```diff
@@ -31,6 +31,7 @@ func NewEtcdCmd() *cobra.Command {
 		Use:   "etcd",
 		Short: "Manage etcd cluster",
 		PreRunE: func(cmd *cobra.Command, args []string) error {
+			// TODO does this ever execute? does this command need the persistentflags?
 			c := CmdOpts(config.GetCmdOpts())
```

Comment:
shouldnot, I think, `k0s etcd` is an umbrella command to group `leave` and `member-list`